lib/net.js

```diff
@@ -256,6 +258,9 @@ function Socket(options) {
 
   // Used after `.destroy()`
   this[BYTES_READ] = 0;
+
+  this[kLastTimeout] = 0;
+  this[kPrevTimeout] = 0;
```

Comment:
IMHO they must be in net because this information is only available in net: https://github.com/nodejs/node/pull/13549/files#diff-e6ef024c3775d787c38487a6309e491dR394